File: src/main/java/me/testaccount666/serversystem/userdata/money/SqliteBankAccount.java
package me.testaccount666.serversystem.userdata.money;

import java.math.BigInteger;
import java.util.UUID;

/**
 * Implementation of AbstractSqlBankAccount for SQLite databases.
 */
public class SqliteBankAccount extends AbstractSqlBankAccount {
    public SqliteBankAccount(UUID owner, BigInteger accountId) {
        super(owner, accountId);
    }
}

File: src/main/kotlin/me/testaccount666/serversystem/userdata/money/SqliteBankAccount.kt
package me.testaccount666.serversystem.userdata.money

import java.math.BigInteger
import java.util.*

/**
 * Implementation of AbstractSqlBankAccount for SQLite databases.
 */
class SqliteBankAccount(owner: UUID, accountId: BigInteger) : AbstractSqlBankAccount(owner, accountId)